src/main/client/src/api/contract/contractRelations.js

```diff
@@ -60,7 +60,7 @@ export const unlinkContract = (
     layerObjectId: number,
     contractLayerId: number,
     contractObjectId: number,
-) => (
+): Promise<boolean> => (
```

Comment:
flow error:
Cannot return `fetch(...).then(...).catch(...)` because undefined [1] is incompatible with boolean [2] in type argument
 862|   log(...data: Array<any>): void;
                                    ^^^^ [1]
   src/api/contract/contractRelations.js:63:12
    63| ): Promise<boolean> => (
                   ^^^^^^^ [2]
   615| declare class Promise<+R> {
                               ^ [3]